sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/DependencyManager.groovy

```diff
@@ -17,12 +18,36 @@ import life.qbic.portal.sampletracking.components.projectoverview.ProjectOvervie
 class DependencyManager {
     VerticalLayout portletView
 
+    private ProjectResourceService projectResourceService
+
     DependencyManager() {
-        ProjectOverviewViewModel viewModel = new ProjectOverviewViewModel()
-        portletView = new ProjectOverviewView(viewModel)
+        setupServices()
+        portletView = setupPortletView()
+    }
+
+    private void setupServices() {
+        this.projectResourceService = new ProjectResourceService()
     }
 
     VerticalLayout getPortletView() {
         return portletView
     }
+
+    private VerticalLayout setupPortletView() {
+        ProjectOverviewView projectOverviewView = createProjectOverviewView()
+        return projectOverviewView
+    }
+
+    /**
+     * Creates a new ProjectOverviewView using
+     * <ul>
+     *     <li>{@link #projectResourceService}</li>
+     * </ul>
+     * @return a new ProjectOverviewView
+     */
+    private createProjectOverviewView() {
+        ProjectResourceService projectResourceService = this.projectResourceService
```

Comment:
I think this variable declaration makes no sense here. There is no interface or such which makes the class attribute different from this variable

```suggestion
```